### PR TITLE
ci: build bootloader emulators

### LINKED ISSUE #3193: Add bootloader emulator builds into CI
They are not being used very often, so it can happen that we break them without noticing - and then they do not work when we need them.

### PR DESCRIPTION
**Pull request by mmilata**
Fixes #3193.

### CODE REVIEW

File: core/SConscript.bootloader_emu [hunk 1]
```diff
@@ -128,7 +128,6 @@ SOURCE_TREZORHAL = [
     'embed/trezorhal/unix/rng.c',
     'embed/trezorhal/unix/usb.c',
     'embed/trezorhal/unix/random_delays.c',
-    'embed/trezorhal/unix/secret.c',
```
Comment (matejcik) [inline core/SConscript.bootloader_emu:131]: isn't this a problem for model R ?

Comment (mmilata) [reply core/SConscript.bootloader_emu:131]: right, it's not added by the board configuration function since this is emulator ...

Comment (TychoVrahe) [reply core/SConscript.bootloader_emu:131]: yeah we need to add this conditionally in sconscript. i was thinking about something like "emulator boards", so far seems a bit like overkill but when adding new models these sconscripts might get messy, so we might eventually need that

Comment (mmilata) [reply core/SConscript.bootloader_emu:131]: 194f517b49df31d3f2176a1cfcb16de02fa356f8

**Review by TychoVrahe — APPROVED** [review]

**Pull request merged**